src/psyclone/tests/.pylintrc

```diff
@@ -25,3 +25,7 @@ disable=print-statement,suppressed-message,duplicate-code,no-self-use,too-many-l
 function-rgx=[a-z_][a-z0-9_]{2,80}$
 
 generated-members=lfric_psyir
+
+[DESIGN]
```

Comment:
I thought (but may be wrong) that we would have inherited this setting from `src/psyclone/.pylintrc` since it is in a parent directory?

Reply:
I didn't know it did that, since it inherits line-length I expect so, removed.